charts/osm/grafana/dashboards/osm-mesh-envoy-details.json

```diff
@@ -1,1145 +1,1251 @@
 {
-  "annotations": {
```

Comment:
What lines did you change/add? It's hard to interpret this diff :)

Reply:
@shalier This is weird... it should not have been that different if you just exported/imported whatever there was.

ie:
https://github.com/openservicemesh/osm/pull/2622/files

Reply:
should be fixed now, the code was auto-formatting